src/tests/system/tests/test_infopipe.py

```diff
@@ -0,0 +1,208 @@
+"""
+Infopipe tests.
+
+:requirement: None
```

Comment:
This needs to be resolved. We want to have a feature test plan/ requirement for this.

Reply:
I don' t understand what is this that needs to be resolved.

Reply:
We can't have :requirement: None. All tests should be linked to appropriate requirements.

Reply:
Finally using `Service/IFP`.